README.md

```diff
@@ -253,6 +253,9 @@ WMCO / Windows nodes does not work with the following products:
 Windows nodes managed by WMCO do not support pulling container images from secure private registries. 
 It is recommended to use images from public registries or pre-pull the images in the VM image.
 
+### Trunk port
+WMCO does not support adding Windows nodes in a networking setup where a trunk port is present. The only supported networking setup is using an access port for carrying the VLAN traffic.
```

Comment:
Is this desriptive enough to describe the problem scenario?
This can be interpreted as a trunk port cannot be present in your cluster network setup at all.
Is it more valid to say that a Windows node cannot be connected to the cluster through a trunk port?

I am not confident enough in my networking knowledge to say for sure.